README.adoc

```diff
@@ -10,38 +10,43 @@
 //  What you'll learn
 // ===============================================================================================
 
 == What is gRPC?
 
-The https://grpc.io/[gRPC^] Remote Procedure Call is a technology that implements remote procedure call (RPC) style APIs with HTTP/2. gRPC uses https://developers.google.com/protocol-buffers/docs/reference/overview[protocol buffers^] to define its routines that include service calls and expected messages. For each service defined in a `.proto` file, gRPC uses the definition to generate the skeleton code for users to implement and extend. Protocol buffers use a binary format to send and receive messages that are faster and lightweight compared to JSON used in RESTful APIs.
+The https://grpc.io/[gRPC^] Remote Procedure Call is a technology that implements remote procedure call (RPC) style APIs with HTTP/2. gRPC uses https://developers.google.com/protocol-buffers/docs/reference/overview[protocol buffers^] to define its routines, which include service calls and expected messages. For each service defined in a `.proto` file, gRPC uses the definition to generate the skeleton code for users to implement and extend. Protocol buffers use a binary format to send and receive messages that is faster and more lightweight than the JSON that is typically used in RESTful APIs.
 
-Protocol buffers allow cross project support through the `.proto` file, as a result gRPC clients and servers are also able to run and communicate with each other on different environments. For example, a gRPC client running in Java codebase is able to call a gRPC server from other https://grpc.io/docs/languages/[supported languages^]. This feature of protocol buffers allows for easier integration between services.
+Protocol buffers allow cross-project support through the `.proto` file. As a result, gRPC clients and servers can run and communicate with each other from different environments. For example, a gRPC client running in Java codebase is able to call a gRPC server from other https://grpc.io/docs/languages/[supported languages^]. This feature of protocol buffers allows for easier integration between services.
 
 == What you'll learn
 
-You will learn how to create gRPC services and their clients by using protocol buffers and how to implement them with Open Liberty. You will use Maven throughout the guide to generate the gRPC stubs and deploy the services and to interact with the running Liberty server.
+You will learn how to create gRPC services and their clients by using protocol buffers and how to implement them with Open Liberty. You will use Maven to generate the gRPC stubs, deploy the services, and to interact with the running Liberty server.
 
 The application that you will build in this guide consists of the `systemproto` model project, the `query` client service, and the `system` server service. The `query` service implements four RESTful APIs by using four different gRPC streaming methods.
 
```

Comment:
added a detail of the four RPCs to orient the reader, please review for accuracy